src/rgw/rgw_rados.cc

```diff
@@ -4641,6 +4641,9 @@ int RGWRados::init_complete()
   data_notifier = new RGWDataNotifier(this);
   data_notifier->start();
 
+  binfo_cache = new RGWChainedCacheImpl<bucket_info_entry>;
+  binfo_cache->init(svc.cache);
```

Comment:
The master commit is just moving two lines. In luminous, the lines being moved are:

```
  binfo_cache = new RGWChainedCacheImpl<bucket_info_entry>;
  binfo_cache->init(this);
```

So why change the second line to `binfo_cache->init(svc.cache);` ?